doc/make_doc

```diff
@@ -4,7 +4,7 @@ tex manual
 ../../../doc/manualindex manual
 tex manual
 mkdir -p ../htm
-../../../etc/convert.pl -c -i -u -n CARAT . ../htm
+../../../etc/convert.pl -c -i -u -n CARATINTERFACE . ../htm
```

Comment:
I think capitalization here doesn't matter, so I'd have a slight preference for using the "proper" one:
```suggestion
../../../etc/convert.pl -c -i -u -n CaratInterface . ../htm
```